linux_sql/README.md

```diff
@@ -1,2 +1,119 @@
-# Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+# Introduction
+As a developer I am tasked with designing, implementing, and demonstrating an MVP (minimal viable product) that helps the LCA (Linux Cluster Administration) team to meet their business needs. The LCA team manages a cluster of 10 nodes/servers all of which are running CentOS 7 and connected through a switch and can communicate through IPV4. 
+
+The LCA team needs an application that records hardware specifications of each node and monitor the resource usage (eg. CPU, memory...) every five minutes. The recorded data is then stored in an RDBMS database and used to generate reports for future resource planning purposes.
+
+#### Tools used:
+<img height="32" width="32" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/docker/docker.png" 
```

Comment:
Please use local content. Upload images in /assets folder and reference them here.